importer_utils.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/python
+# -*- coding: utf-8  -*-
+import datetime
+import json
+import re
+import pywikibot
+import os
+from wikidataStuff.WikidataStuff import WikidataStuff as wds
+
+site_cache = {}
+
+DATA_DIRECTORY = "data"
+
+
+def load_json(filename):
+    try:
+        with open(filename) as f:
+            try:
+                return json.load(f)
+            except ValueError:
+                print("Failed to decode file {}.".format(filename))
```

Comment:
This is just for information since it has never actually happened. So I've never had a practical reason to see what happens in such a case.